include/ecs/Flag.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace arch::ecs::_details { // NOLINT
+
+/// @brief Checks if component was marked as flag-component
+template<class T>
+concept FlagComponent = T::flag_component or T::flagComponent or T::FlagComponent;
```

Comment:
please use || (unless we agreed to using or in concepts)

Reply:
as far as I remember use of alternative operators (or lack thereof) [is not mandatory](https://github.com/AGH-Code-Industry/arch-org/blob/master/documents/rules/code_style.md#zach%C4%99cenie-do-u%C5%BCycia-operator%C3%B3w-alternatywnych)
I personally find logic operators to be more readable in this way